packages/data-access/src/utils.ts

```diff
@@ -16,3 +17,20 @@ export function isStorageError(
     (maybeStorageError as StorageError).name === 'StorageError'
   );
 }
+
+export function toBlobName({
+  projectName,
+  version,
+  moduleName,
+  realTime,
+}: ReportIdentifier) {
+  const slug = [
+    projectName,
+    version,
+    moduleName,
+    realTime ? 'real-time' : realTime,
+  ]
+    .filter(Boolean)
+    .join('/');
+  return encodeKey(slug);
```

Comment:
What will happen when I create a branch named `real-time`? This will be a naming clash, right? Wouldn't it be better to use a separate Container for real-time?

Reply:
The following happens:
```js
    it('what happens when real-time is used as a branch name', () => {
      expect(
        toBlobName({
          projectName: 'abc',
          version: 'real-time',
          moduleName: 'def',
          realTime: true
        })
        ).to.eq('abc;real-time;def;real-time') // passes
    });
```

I don't think this will cause any naming clashes, since real-time adds a fourth part to the blobname.